elements/urth-core-channels/urth-core-channels.html

```diff
@@ -105,9 +105,11 @@
             console.debug('urth-core-channels onModelChange', options);
 
             var changes = options.changed;
-            Object.keys(changes).forEach(function(change){
+            Object.keys(changes).filter(function(change){
+                return !(change === "status");
```

Comment:
I also don't like the filtering happening here. It feels like if we still want to use the `sync` message to carry the error, we should do the filtering on jupyter-widget-behavior._onModelChange()